screens/home.js

```diff
@@ -1,208 +1,222 @@
-     }
-
-    onQuickReply = replies => {
-       const createdAt = new Date()
-       if (replies.length === 1) {
-         this.onSend([
-           {
-             createdAt,
-             _id: Math.round(Math.random() * 1000000),
-             text: replies[0].title,
-             user,
-           },
-         ]);
-         this.determineResponse(replies[0]);
-       } else if (replies.length > 1) {
-         this.onSend([
-           {
-             createdAt,
-             _id: Math.round(Math.random() * 1000000),
-             text: replies.map(reply => reply.title).join(', '),
-             user,
-           },
-         ]);
-       } else {
-         console.warn('replies param is not set correctly')
-       }
-     }
-
-     determineResponse = reply => {
-       const createdAt = new Date()
-       console.log(reply.title);
-       if (reply.value === 'no'){
-         this.onSend([
-           {
-             createdAt,
-             _id: Math.round(Math.random() * 1000000),
-             text: "have a nice day :(",
-             otherUser,
-           },
-         ])
-       }
-       if (reply.value === 'yes_picture'){
-         this.onSend([
-           {
-             createdAt,
-             _id: Math.round(Math.random() * 1000000),
-             text: "jokes i have no pics",
-             otherUser,
-           },
-         ])
-       }
-       if (reply.value === 'yes'){
-         this.onSend([
-           {
-
-                   _id: 2,
-                   text: 'Awesome, Did you sleep last night?',
-                   createdAt: new Date(),
-                   quickReplies: {
-                       type: 'radio', // or 'checkbox',
-                       keepIt: true,
-                       values: [
-
-                         {
-                           title: 'For a bit',
-                           value: 'yes_picture',
-                         },
-                         {
-                            title: 'Yes 😋 ',
-                            value: 'yes',
-                         },
-                         {
-                           title: 'Nope. Was up all night😞 ',
-                           value: 'no',
-                         },
-                       ],
-                     },
-                     user: {
-                        _id: 2,
-                        name: 'React Native',
-                   },
-
-
-           },
-         ])
-       }
-
-     }
-    renderQuickReplySend = () => <Text>{' custom send =>'}</Text>
-    renderInputToolbar(props){
-    if(this.state.toolbar){
-    return(
-    <InputToolbar {...props} />
-    );
+            _id: 2,
+            name: "React Native"
+          }
+        }
+      ]);
     }
+  };
+  renderQuickReplySend = () => <Text>{" custom send =>"}</Text>;
+  renderInputToolbar(props) {
+    if (this.state.toolbar) {
+      return <InputToolbar {...props} />;
     }
-    render() {
+  }
+  renderFooter = () => {
+    if (this.state.typingText) {
```

Comment:
Tends to be bad practice for `this.state.<something>` if that "something" occurs more than once. 
Usually it'll be like:
`const { typingText } from this.state;`
That way, it reduces the likelihood of the entire file flooded with `this.state`.
Change in the future if you want but we're not going to be that strict here.